src/main/java/interpol/entity/Order.java

```diff
@@ -0,0 +1,155 @@
+package interpol.entity;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "order")
+public class Order {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    // Связь OneToOne на Country
+//    @Column(name = "country_id")
+//    private int country_id;
+    @ManyToOne(fetch = FetchType.EAGER, cascade = {CascadeType.MERGE, CascadeType.PERSIST})
+    @JoinColumn(name = "country_id", nullable = false)
+    private Country country;
+
+    // Связь ManyToOne на User
+//    @Column(name = "user_id")
+//    private int user_id;
+    @ManyToOne(fetch = FetchType.EAGER, cascade = {CascadeType.MERGE, CascadeType.PERSIST})
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
+    //Связь OneToOne на OrderDetail
+//    @Column(name = "order_detail_id")
+//    private int order_detail_id;
+    @OneToOne(optional = false)
+    @JoinColumn(name = "order_detail_id", nullable = false)
+    private OrderDetail orderDetail;
```

Comment:
Both `OrderDetail` and `OrderResult` should have the same structure, but now in `"order"` table tou have order_detail_id field but now order_result_id field.
Lets rework OrderDetail class (and table) to have foreign key to `order` (instead of having this foreign key in order table)